src/components/pages/Home/Offerers/CreationLinks.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+
+import { isAPISireneAvailable } from 'store/selectors/data/featuresSelectors'
+import { UNAVAILABLE_ERROR_PAGE } from 'utils/routes'
+
+const CreationLink = ({ hasPhysicalVenue, hasVirtualOffers, offererId }) => {
+  const isVenueCreationAvailable = useSelector(isAPISireneAvailable)
+
+  const venueCreationUrl = isVenueCreationAvailable
+    ? `/structures/${offererId}/lieux/creation`
+    : UNAVAILABLE_ERROR_PAGE
+
+  const renderCreationLinks = insideCard => (
+    <div className="actions-container">
+      <Link
+        className={insideCard ? 'primary-link' : 'secondary-link'}
+        to={venueCreationUrl}
+      >
+        {!hasPhysicalVenue ? 'Créer un lieu' : 'Ajouter un lieu'}
+      </Link>
+
+      {!hasVirtualOffers && (
+        <Link
+          className="secondary-link"
+          to={`/offres/creation?structure=${offererId}`}
+        >
+          {'Créer une offre numérique'}
+        </Link>
+      )}
+    </div>
+  )
+
+  const renderCard = () => (
+    <div
+      className="h-card"
+      data-testid="offerers-creation-links-card"
+    >
+      <div className="h-card-inner">
+        <h3 className="h-card-title">
+          {'Lieux'}
+        </h3>
+
+        <div className="h-card-content">
+          <p>
+            {'Avant de créer votre première offre physique vous devez avoir un lieu'}
+          </p>
+          {renderCreationLinks(true)}
```

Comment:
Je trouve ça plus compréhensible que de donner directement `true` en paramètre
```suggestion
          {renderCreationLinks({ insideCard: true })}
```